src/main/resources/META-INF/frontend/styles/styles.css

```diff
@@ -0,0 +1,72 @@
+.fc-dtrp-hoverable:hover, .fc-dtrp-selected {
```

Comment:
Rename this file as `fc-date-time-range-picker.css` styles so that it doesn't conflict with `styles.css` in applications that use the add-on.